rotkehlchen/chain/ethereum/trades.py

```diff
@@ -0,0 +1,245 @@
+from __future__ import annotations
```

Comment:
@vnavascues I see why you do this. But it's not needed. From python 3.10 [it will be the default](https://docs.python.org/3/whatsnew/3.7.html#pep-563-postponed-evaluation-of-annotations), but for now I would like to avoid importing stuff `from __future__` unless it's absolutely needed.

And this is not needed. You can see how we do it in other parts of the code. I am going to change it now as I am working on the PR, but just to let you know you can avoid it by putting a string representation of the type annotation like so:

`def get_amm_trade_from_db_tuple() -> 'AMMTrade':`

And if you do that mypy works fine.